jobs/enqueuer.go

```diff
@@ -0,0 +1,119 @@
+package jobs
+
+import (
+	"fmt"
+
+	"github.com/gocraft/work"
+	"github.com/gomodule/redigo/redis"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	ComponentName = "jobs-enqueuer"
+)
+
+const (
+	defaultPoolSize  = 5
+	defaultNamespace = "foundation_jobs_worker"
+)
+
+type Component struct {
+	Enqueuer *work.Enqueuer
+
+	url       string
+	namespace string
+	poolSize  int
+	logger    *logrus.Entry
+}
+
+// ComponentOption is an option to `Component`.
+type ComponentOption func(*Component)
+
+// WithLogger sets the logger for the JobsEnqueuer component
+func WithLogger(logger *logrus.Entry) ComponentOption {
+	return func(c *Component) {
+		c.logger = logger.WithField("component", c.Name())
+	}
+}
+
+// WithURL sets the database URL for the JobsEnqueuer component.
+func WithURL(url string) ComponentOption {
+	return func(c *Component) {
+		c.url = url
+	}
+}
+
+// WithPoolSize sets the pool size for the JobsEnqueuer component.
+func WithPoolSize(poolSize int) ComponentOption {
+	return func(c *Component) {
+		c.poolSize = poolSize
```

Comment:
I think we should get this value from the `Config`, as we do for the `Database.Pool`

Reply:
added to config